cms/.env.example

```
@@ -1,3 +1,7 @@
+STRAPI_DISABLE_UPDATE_NOTIFICATION=false
+STRAPI_TELEMETRY_DISABLED=true
+NODE_ENV=local
```

I guess change this to development

```suggestion
NODE_ENV=development
```